app/templates/cli.js

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env node
+'use strict';
+var meow = require('meow');
+var <%= camelModuleName %> = require('./');
+
+var cli = meow({
+	help: [
+		'Usage',
+		'  $ <%= moduleName %> [object]',
+		'',
+		'Examples',
+		'  $ <%= moduleName %>',
+		'  unicorns & rainbows',
+		'',
+		'  $ <%= moduleName %> ponies',
+		'  ponies & rainbows'
+	].join('\n')
+});
+
+console.log(<%= camelModuleName %>(cli.input[0] || 'unicorns'));
```

Comment:
and can you pass in the flags to the second parameter: https://github.com/sindresorhus/generator-nm/blob/master/app/templates/readme.md#foo

```js
console.log(chalk.magenta.bold(<%= camelModuleName %>(cli.input[0] || 'unicorns', cli.flags)));
```